profiles/package.mask

```diff
@@ -33,6 +33,12 @@
 
 #--- END OF EXAMPLES ---
 
+# Volkmar W. Pogatzki <gentoo@pogatzki.net> (2022-04-22)
+# Blocks removal of vulnerable dev-java/guava:20, bug #833309.
+# Removal in 30 days.
+dev-util/checkstyle
+app-emacs/jde
```

Comment:
What about app-xemacs/jde?

Reply:
`app-xemacs/jde` doesn't seem to appear in reverse dependencies.
Just reverted to draft mode, will followup later

Reply:
I think we can ignore app-xemacs/jde then. And @ulm from gnu-emacs@ just ack'ed the tree cleaning of app-emacs/jde.

Reply:
@Flowdalic Do you still need 2 separate PRs to proceed, or is enough to split into separate commits?

Reply:
Na, I think this is good to go as is. Will merge once you remove the draft status. :)

Reply:
Done.